routers/web/user/code.go

```diff
@@ -19,17 +20,16 @@ const (
 
 // CodeSearch render user/organization code search page
 func CodeSearch(ctx *context.Context) {
+	user.RenderProfileBigAvatar(ctx)
```

Comment:
The name and usage seems not ideal.

* The name: it doesn't "render", it only prepares some data
* If the response has been written in it, the caller shouldn't continue (by checking `Written`)

Reply:
Correct, and I am even in favour of changing the naming pattern, but when I see https://github.com/go-gitea/gitea/pull/25108/files/6c990c4da74a329f700ffffaa5ddfa1d6195fd7b#diff-f4279417070a8e33829c338abeb42877500377f490abb1495ae6357d50b6a765R92 in the same function doing something like that and implementation looks like this https://github.com/go-gitea/gitea/blob/e24f651c86a5168da8181f2c75f5f99b56561224/routers/web/shared/user/header.go#L13,
 I don't want to break the pattern, if you are suggesting an explicit change in function, that I can make that change, 
 
 In short maintainability is the real concern here.

Reply:
Well, the old code is buggy, so I won't say it is a must now.